src/ducks/apps/components/UninstallModal.jsx

```diff
@@ -40,6 +40,11 @@ export class UninstallModal extends Component {
 
   render() {
     const { isUninstalling, dismissAction, t, uninstallError } = this.props
+    const linkedAppError =
+      uninstallError &&
+      uninstallError.message === 'A linked OAuth client exists for this app'
```

Comment:
Is it safe to get an error type based on a description here? Do we have a specific code here or something else with less chance to change

Reply:
We don't have a specific code for this error. We have `Bad Request` code, but it can be used for other errors.